README.md

```diff
@@ -172,112 +163,144 @@ specify the class of the instance we want to access.
+      return name;
+   }
 
-       public void setName(String name) {
-           this.name = name;
-       }
+   public void setName(String name) {
+      this.name = name;
    }
-   ```
+}
+```
 
-2. Use the `@ComponentScan` annotation over the configuration class and tell it
-   where to look for classes annotated with stereotype annotations.
+2. To look for component classes, we can use the `@ComponentScan` annotation over
+   the configuration class to tell Spring where to look for classes annotated with
+   stereotype annotations. We'll pass in the `domain` package to tell Spring to
+   scan that as the base package.
 
-   ```java
+```java
    package config;
 
    import org.springframework.context.annotation.ComponentScan;
    import org.springframework.context.annotation.Configuration;
 
    @Configuration
-   @ComponentScan(basePackages = "main")
+   @ComponentScan(basePackages = "com.example.demo.domain")
    public class MyConfig {
-
+       
    }
-   ```
+```
+With the code above, we have now told Spring which classes to add to its context
+and where to find these classes through the `@ComponentScan` annotation. This in
+turn tells Spring to recognize our Java code!
 
 Spring will create the instance for us which means we can’t set the `name`
 property on the `Dog` instance during instantiation. Instead, we will get the
-`Dog` instance after Spring creates it and set the `name` property. Update the
-`Main` class code:
+`Dog` instance after Spring creates it and sets the `name` property. Update the
+`Main` class code to prove that Spring has created and added the bean in its
+context:
 
 ```java
-package main;
+package com.example.demo;
 
-import config.MyConfig;
+import com.example.demo.config.MyConfig;
+import com.example.demo.domain.Dog;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 
-public class Main {
-    public static void main(String[] args) {
-        // create context
-        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(MyConfig.class);
-        // access bean
-        Dog luna = context.getBean(Dog.class);
-        luna.setName("Luna");
-        System.out.println(luna);
-    }
+@SpringBootApplication
+public class DemoApplication {
+   public static void main(String[] args) {
+      SpringApplication.run(DemoApplication.class, args);
+
+      // create Spring context instance
+      AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(MyConfig.class);
+
+      // access the bean
+      Dog luna = context.getBean(Dog.class);
+      luna.setName("Luna");
+      System.out.println(luna);
+   }
 }
 ```
 
 Running the `main` method will print out the `Dog` instance in the Spring
 context. The stereotype annotations are primarily used over the `@Bean`
 annotations since it is more simple.
```

Comment:
I don't think this last sentence is completely correct. The stereotype and bean annotations have different purposes (I think the table below does a good job explaining that).

Reply:
Agreed. Removed this sentence.